### ISSUE TITLE
Accessibility “bullets hollow” should be “more options”

### ISSUE THREAD

**Issue by BasThomas** [GitHawkApp/GitHawk#2673]
![GitHawk Upload by BasThomas](https://i.imgur.com/8gtdLlq.jpg)

<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.27.1 (1546622353)
Device: iPhone X (iOS 12.1.4)
TestFlight: true
</pre>
</details>

**Comment by wayni208** [comment]
I can look into this. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
